test/wrappers/here_test.rb

```diff
@@ -184,4 +184,11 @@ def test_distance_should_define_row_number
       assert_equal(obj[:max_dsts], RouterWrapper::HERE_TRUCK.send(:max_dsts, obj[:distance], [*1..100], obj[:options]))
     end
   end
+
+  def test_no_content
```

Comment:
`test_nil_if_unreachable` | `test_prevent_no_content_in_case_unreachable`

Reply:
Done. I prefer `test_prevent_no_content_in_case_unreachable` because it is speaks functionality.